app/test_split_integer.py

```diff
@@ -2,20 +2,39 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 32
+    number_of_parts = 6
+    output_list = split_integer(value, number_of_parts)
+    assert sum(output_list) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 32
+    number_of_parts = 6
+    output_list = split_integer(value, number_of_parts)
+
+    for element in output_list:
+        assert element == value // number_of_parts
```

Comment:
The assertion logic here is incorrect. The use of `value // number_of_parts` assumes that the division will always be exact, which might not be the case. Instead, you should check if all elements in `output_list` are equal to each other, which would confirm that the list is split into equal parts when `value` is divisible by `number_of_parts`.